app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/controllers/agent/BusinessStartDateController.scala

```diff
@@ -94,6 +100,15 @@ class BusinessStartDateController @Inject()(mcc: MessagesControllerComponents,
     }
```

Comment:
the flow mentioned in the story says that from this page we then go to BusinessTradeName rather than BusinessName